demos/tensorflow/quickstart.py

```diff
@@ -0,0 +1,93 @@
+import numpy as np
+import tensorflow as tf
+
+from coalescenceml.pipeline import pipeline
+from coalescenceml.step import step, Output
+from coalescenceml.integrations.constants import TENSORFLOW
+from coalescenceml.integrations.tensorflow.step import (
+    TFClassifierTrainConfig,
+    TFClassifierTrainStep,
+)
+
+@step
+def importer() -> Output(
```

Comment:
Ask Zander for spiral data code generator OR get it from onboarding material and use it for this data generation